.git-blame-ignore-revs

```diff
@@ -0,0 +1 @@
+ee4f520047dab4d01b58b106cb940f0a6abad1aa
```

Comment:
shouldnt this commit sha be updated?

Reply:
i dont see the formatting commit anywhere

Reply:
Yeah, it should be updated